src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/collectionsgrid/viewmodel/CollectionsGridViewModel.kt

```diff
@@ -2,26 +2,21 @@ package org.wycliffeassociates.otter.jvm.app.ui.collectionsgrid.viewmodel
 
 import com.github.thomasnield.rxkotlinfx.observeOnFx
 import com.github.thomasnield.rxkotlinfx.toObservable
-import javafx.beans.property.SimpleBooleanProperty
-import javafx.beans.property.SimpleObjectProperty
 import javafx.collections.FXCollections
 import javafx.collections.ObservableList
 import org.wycliffeassociates.otter.common.data.model.Collection
-import org.wycliffeassociates.otter.common.data.model.Content
-import org.wycliffeassociates.otter.common.domain.content.AccessTakes
-import org.wycliffeassociates.otter.jvm.app.ui.collectionsgrid.view.CollectionGridStyles
+import org.wycliffeassociates.otter.common.data.model.ResourceMetadata
 import org.wycliffeassociates.otter.jvm.app.ui.collectionsgrid.view.CollectionTabPane
 import org.wycliffeassociates.otter.jvm.app.ui.inject.Injector
 import org.wycliffeassociates.otter.jvm.app.ui.mainscreen.view.MainScreenStyles
-import tornadofx.*
```

Comment:
According to the tornadofx documentation we should be importing `tornadofx.*` in all tornadofx-related files https://edvin.gitbooks.io/tornadofx-guide/content/part1/3.%20Components.html

Reply:
Oh, I think that's so auto-complete can find tfx extension methods on non-tfx classes. This should be changed back.